src/core/lib/channel/connected_channel.cc

```diff
@@ -908,11 +924,34 @@ const grpc_channel_filter kServerEmulatedFilter =
     MakeConnectedFilter<nullptr>();
 #endif
 
-bool TransportSupportsPromiseBasedCalls(const ChannelArgs& args) {
+const grpc_channel_filter kServerPromiseBasedTransportFilter = {
```

Comment:
Please add a comment explaining that this is a no-op and will never be called, because the v3 stack will instead pass control to the CallInitiator held by the transport.